### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by vixentael**
Adding issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Acra configuration files**
+For AcraServer: 
+    - [ ] configuration file or CLI params that you use to start AcraServer; 
+    - [ ] `encryptor_config.yaml` if used.
+For AcraTranslator: 
+    - [ ] configuration file or CLI params that you use to start AcraTranslator.
+
+**Environment (please complete the following information):**
+- Acra version: [e.g. 0.90.0]
```
Comment (Lagovas) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: would be useful to specify database type and version. PostgreSQL 11, MySQL 7, mariadb 10, or some cloud based with supported wire protocol.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[ISSUE]"
+labels: question
+assignees: Lagovas
```
Comment (Lagovas) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: okaaaaay

**Review by Lagovas — APPROVED** [review]
lgtm with note about databases

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (vixentael) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
- Database server and its version: [e.g. PostgreSQL 11, MySQL 7 ..]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+**Acra configuration files**
+For AcraServer: 
+    - [ ] configuration file or CLI params that you use to start AcraServer; 
+    - [ ] `encryptor_config.yaml` if used.
+For AcraTranslator: 
+    - [ ] configuration file or CLI params that you use to start AcraTranslator.
+
+**Environment (please complete the following information):**
+- Database server and its version: [e.g. PostgreSQL 11, MySQL 7 ..]
```
Comment (vixentael) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
- Acra version: [e.g. 0.90.0]
- Database server and its version: [e.g. PostgreSQL 11, MySQL 7 ..]
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (vixentael) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: good idea, done

**Pull request merged**